en/docs/setup/migrating-to-5110.md

```diff
@@ -138,9 +138,9 @@ Once all the above prerequisites have been met, follow the instructions given be
 
         Make sure to point the internal keystore to the keystore that is copied from the previous WSO2 Identity Server version. The primary keystore can be pointed to a keystore with a certificate that has a strong RSA key.
 
-4.  To migrate the super tenant, copy the content from `<OLD_IS_HOME>/repository/deployment/server` directory to the `<NEW_IS_HOME>/repository/deployment/server` directory. 
+4.  To migrate the super tenant, copy the content (except the webapps folder) from `<OLD_IS_HOME>/repository/deployment/server` directory to the `<NEW_IS_HOME>/repository/deployment/server` directory. 
 
-    If you have created tenants in the previous WSO2 Identity Server
+    If you have customized the web apps in the previous WSO2 Identity Server version, those changes should be manually applied to the relevant files with careful inspection. If you have created tenants in the previous WSO2 Identity Server
```

Comment:
I think the content here is a bit jumbled. It should look like the following.

![image](https://user-images.githubusercontent.com/33062368/195568647-d1a84ea0-f110-4f88-9924-d758e3c2c51f.png)

If there are custom webapps in the previous IS, those should be made compatible with the newer IS. If the existing webapps were customized in the previous IS, then those changes should be added to the relevant files in the new IS with careful inspection.